src/webpack-config.ts

```diff
@@ -8,16 +8,19 @@ const SVG_ICONS_PATH_REGEX = /icons\/.*\.svg$/;
 const IMAGE_PATH_REGEX = /\.(jpe?g|png|gif|svg)$/;
 const LIBRARY_REGEX = /node_modules\/@shopify\/checkout-ui-react/;
 
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const {
```

Comment:
This should be fine to import (I didn't do it with the other ones because I didn't want to bother finding type definitions)